### DISCUSSION TITLE
Quick App No File Issue

### DISCUSSION THREAD

**Discussion by primewaydesign** [leokhoa/laragon#717] [General]
Hi,

I hope someone can help.

On Windows 10 Pro I always face this issue:

![laragon_2024-03-08_190759](https://github.com/leokhoa/laragon/assets/16689902/9e83e1e9-8721-46f3-8f4f-1a49b2fb32ec)

Any ideas of how resolve this?
I'm using Laragon version 5.0.0.210523.

**Comment by leokhoa** [comment]
@primewaydesign : Why do you blur the path?
On the original image, it is truncated:
<img width="152" alt="image" src="https://github.com/leokhoa/laragon/assets/9049575/6301e6e3-3a67-41c8-bc3c-a678d37b8839">

Does your path contain something special?

**Reply by primewaydesign** [reply]
![laragon_2024-03-08_190759](https://github.com/leokhoa/laragon/assets/16689902/7d200927-4dcd-4751-bc5c-75b3adc9c395)
